var/spack/repos/builtin/packages/python/package.py

```diff
@@ -369,845 +339,826 @@ def flag_handler(self, name, flags):
-    # Set up environment to make install easy for python extensions.
-    # ========================================================================
+        Returns:
+            str: platform-independent site-packages directory
+        """
+        prefix = self.config_vars["base"] + os.sep
+        path = self.config_vars["purelib"]
+        if path.startswith(prefix):
+            return path.replace(prefix, "")
+        return os.path.join("lib", "python{}".format(self.version.up_to(2)), "site-packages")
 
     @property
-    def command(self):
-        """Returns the Python command, which may vary depending
-        on the version of Python and how it was installed.
+    def include(self):
+        """Directory for non-platform-specific header files.
+
+        Exact directory depends on platform/Python version/ABI flags. Examples include:
 
-        In general, Python 3 only comes with a ``python3`` command. However, some
-        package managers will symlink ``python`` to ``python3``, while others
-        may contain ``python3.11``, ``python3.10``, and ``python3.9`` in the
-        same directory.
+        * ``include/pythonX.Y`` on most POSIX systems
+        * ``include/pythonX.Yd`` for debug builds
+        * ``include/pythonX.Ym`` for malloc builds
+        * ``include/pythonX.Yu`` for wide unicode builds
+        * ``include`` on macOS with framework Python
+        * ``Include`` on Windows
 
         Returns:
-            Executable: the Python command
+            str: platform-independent header file directory
         """
-        # We need to be careful here. If the user is using an externally
-        # installed python, several different commands could be located
-        # in the same directory. Be as specific as possible. Search for:
-        #
-        # * python3.11
-        # * python3
-        # * python
-        #
-        # in that order if using python@3.11.0, for example.
-        version = self.spec.version
-        for ver in [version.up_to(2), version.up_to(1), ""]:
-            if sys.platform != "win32":
-                path = os.path.join(self.prefix.bin, "python{0}".format(ver))
-            else:
-                path = os.path.join(self.prefix, "python{0}.exe".format(ver))
-            if os.path.exists(path):
-                return Executable(path)
+        prefix = self.config_vars["installed_base"] + os.sep
+        path = self.config_vars["include"]
+        if path.startswith(prefix):
+            return path.replace(prefix, "")
+        return os.path.join("include", "python{}".format(self.version.up_to(2)))
 
-        else:
-            # Give a last try at rhel8 platform python
-            if self.spec.external and self.prefix == "/usr" and self.spec.satisfies("os=rhel8"):
-                path = os.path.join(self.prefix, "libexec", "platform-python")
-                if os.path.exists(path):
-                    return Executable(path)
```

Comment:
Can you please double check this file, comparing it to develop? This for instance was deleted in the current PR.